CHANGELOG.md

```diff
@@ -5,6 +5,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+## Added
+- Add new functionality `async postGetValidate()`
```

Comment:
:thinking: Es redundante poner "Add new functionality" en la sección "Added"

Sugiero

`- postGetValidate method to validate registry after getting it`

o algo asi (con un mejor ingles)